common/logisticspipes/utils/InventoryUtil.java

```diff
@@ -58,21 +58,24 @@ public int itemCount(ItemIdentifier item) {
 	}
 
 	@Override
-	public Map<ItemIdentifier, Integer> getItemsAndCount() {
+	public Map<ItemIdentifier, Integer> getItemsAndCount()
```

Comment:
Nothing in this class has actually been changed functionality wise as far as I can see, so please revert